core/client/src/main/java/alluxio/client/block/stream/BlockOutStream.java

```diff
@@ -117,42 +112,38 @@ public long remaining() {
   }
 
   @Override
-  public void cancel() throws IOException {
+  public void cancel() {
     if (mClosed) {
       return;
     }
-    Throwable throwable = null;
+    Exception exception = null;
     try {
       mOutStream.cancel();
-    } catch (Throwable e) {
-      throwable = e;
+    } catch (Exception e) {
+      exception = e;
     }
     try {
       mBlockWorkerClient.cancelBlock(mBlockId);
-    } catch (Throwable e) {
-      throwable = e;
+    } catch (Exception e) {
+      exception = e;
     }
 
-    if (throwable == null) {
+    if (exception == null) {
       mClosed = true;
       return;
     }
 
     try {
       mCloser.close();
-    } catch (Throwable e) {
+    } catch (Exception e) {
```

Comment:
Because we expect `close` to throw unchecked exceptions now? It might be worth updating our utility methods to swallow those as well, since that should be in line with the original purpose of the method.